inductiva/molecules/scenarios/protein_solvation/post_processing.py

```diff
@@ -26,24 +28,28 @@ def __init__(self, sim_output_path: Path = None):
 
     def render_interactive(self,
                            representation: Literal["cartoon", "ball+stick",
-                                                   "line", "point",
+                                                   "line", "point", "surface",
                                                    "ribbon"] = "ball+stick",
+                           selection: str = "protein",
                            use_compressed_trajectory: bool = False,
                            add_backbone: bool = True):
         """
         Render the simulation outputs in an interactive visualization.
         Args: 
             representation: The protein representation to use for the 
             visualization.
+            selection: The selection to use for the representation. Check 
+            https://nglviewer.org/ngl/api/manual/usage/selection-language.html 
+            for details.
             add_backbone: Whether to add the protein backbone to the 
             visualization.
             use_compressed_trajectory: Whether to use the compressed 
             trajectory or the full precision one.
             """
         universe = self.construct_universe(use_compressed_trajectory)
```

Comment:
There is only need for a compressed trajectory here? Iirc before I was using a trajectory and a apdb file, why does it only need one rn?

Reply:
Ah, inside the construct_universe function it reads the topology file!